app/javascript/components/ActionItemForm.jsx

```diff
@@ -0,0 +1,403 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import validator from 'validator';
+import MUIRichTextEditor from 'mui-rte';
+import { makeStyles } from '@material-ui/core/styles';
+import { apiPost, apiPatch } from 'utils/axios';
+import {
+    Button, 
+    TextField,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    MenuItem,
+    Switch, 
+    Grid, 
+    Paper
+} from '@material-ui/core';
+import { createMuiTheme, MuiThemeProvider} from '@material-ui/core/styles'
+import { EditorState, convertToRaw } from 'draft-js';
+
+
+const styles = {
```

Comment:
We probably want to transfer this styling stuff into its own `styles.js` file eventually inside of a directory called ActionItemForm